### PR TITLE
SSPulltoRefresh Implementation

### PR DESCRIPTION
**Pull request by mansivsimform**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: SSCustomPullToRefresh/Views/SpinnerAnimationView.swift [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        // If we're refreshing and the animation is not playing, then play the animation
+        if (self.refreshControl!.isRefreshing && !self.isRefreshAnimating) {
+            self.animateRefreshView()
+        }
+        
+    }
+   
+    public func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        // Distance the table has been pulled >= 0
+        pullDistance = max(0.0, -self.refreshControl!.frame.origin.y)
```
Comment (bhargavbajani-simformsolutions) [inline SSCustomPullToRefresh/Views/SpinnerAnimationView.swift]: do not use force cast

File: SSCustomPullToRefresh/Views/SpinnerAnimationView.swift [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        self.imgSpinnerView.frame = spinnerFrame
+        
+        // Set the refreshBounds view's frames
+        refreshBounds.size.height = pullDistance
+        
+        self.backgroundColorView.frame = refreshBounds
+        self.refreshBaseView.frame = refreshBounds
+
+        // If we're refreshing and the animation is not playing, then play the animation
+        if (self.refreshControl!.isRefreshing && !self.isRefreshAnimating) {
```
Comment (bhargavbajani-simformsolutions) [inline SSCustomPullToRefresh/Views/SpinnerAnimationView.swift]: do not use force cast

Comment (bhargavbajani-simformsolutions) [reply SSCustomPullToRefresh/Views/SpinnerAnimationView.swift]: Please check in full code

File: SSCustomPullToRefresh/Views/SpinnerAnimationView.swift (hunk 1 above)
Comment (mobile-simformsolutions) [reply SSCustomPullToRefresh/Views/SpinnerAnimationView.swift]: Done

File: SSCustomPullToRefresh/Views/SpinnerAnimationView.swift (hunk 2 above)
Comment (mobile-simformsolutions) [reply SSCustomPullToRefresh/Views/SpinnerAnimationView.swift]: Done

**Pull request merged**